### PR TITLE
Direct paging improvements

### PR DESCRIPTION
**Pull request by vadimkerr**
# What this PR does

- Deprecates `/oncall` Slack command in favour of `/esalate` (direct paging) + fixes a regression bug in both commands
- Unifies direct paging UX across Slack & Web UI (or at least makes an attempt to make things more similar). Kudos to @iskhakov for all the great work on this recently!
- A bunch of minor changes that hopefully make direct paging more usable
- TODO: documentation updates will be added in a separate PR

### No issues scenario

Slack:

Screenshot 2023-07-14 at 23 53 11

Web UI: 

Screenshot 2023-07-14 at 23 52 25

### Not configured scenario

Slack:

Screenshot 2023-07-14 at 23 45 22

Web UI:

Screenshot 2023-07-14 at 23 47 31

### `/oncall` deprecation warning

Screenshot 2023-07-17 at 10 31 56

## Which issue(s) this PR fixes

https://github.com/grafana/oncall/issues/2442

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/alerts/paging.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _trigger_alert(
         deleted_at=None,
         defaults={
             "author": from_user,
-            "verbal_name": "Direct paging",
+            "verbal_name": f"Direct paging ({team.name if team else 'No'} team)",
```
Comment (iskhakov) [inline engine/apps/alerts/paging.py:45]: Why do we hardcode team name in the name of Integration? What if it changes

File: grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx [hunk 2]
```diff
@@ -122,41 +122,32 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
             <LoadingPlaceholder text="Loading..." />
           ) : selectedTeamDirectPaging ? (
             <VerticalGroup>
-              <Label>Team will be notified according to the integration settings:</Label>
+              <Label>Integration to be used for notification</Label>
               <ul className={cx('responders-list')}>
                 <li>
                   <HorizontalGroup justify="space-between">
                     <HorizontalGroup>
-                      {escalationChainsExist && (
```
Comment (iskhakov) [inline grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx:130]: why is this removed?

File: engine/apps/alerts/paging.py (hunk 1 above)
Comment (vadimkerr) [reply engine/apps/alerts/paging.py:45]: https://github.com/grafana/oncall/pull/2537#discussion_r1264210758

File: grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx (hunk 2 above)
Comment (vadimkerr) [reply grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx:130]: This is related to https://github.com/grafana/oncall/pull/2537#discussion_r1264221326, the PR udpates which integrations are considered "bad" or non-configured, so now this tooltip doesn't convey any additional information, so I removed it.

**Review by iskhakov — APPROVED** [review]

**Review by Ukochka — APPROVED** [review]
